src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.json

```diff
@@ -71,12 +66,9 @@
         "maccatalyst"
       ],
       "packs": [
-          "Microsoft.Maui.Core.Ref.maccatalyst",
-          "Microsoft.Maui.Core.Runtime.maccatalyst",
-          "Microsoft.Maui.Controls.Ref.maccatalyst",
-          "Microsoft.Maui.Controls.Runtime.maccatalyst",
-          "Microsoft.Maui.Essentials.Ref.maccatalyst",
-          "Microsoft.Maui.Essentials.Runtime.maccatalyst"
+          "Microsoft.Maui.Core",
+          "Microsoft.Maui.Controls",
+          "Microsoft.Maui.Essentials"
```

Comment:
I think you can remove these 3 packs from all the `maui-*` platform workloads, because they all extend `maui-core`?

It wouldn't cause an isuse, just makes things cleaner.